_data/catalog/widgets.json

```diff
@@ -547,6 +547,56 @@
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
   {
+    "name": "CupertinoNavigationBar",
```

Comment:
This file is generated, see https://github.com/flutter/website/blob/master/README.md

The current scheme, where the samples live in the Flutter github repo and the screenshots are generated automatically, may have outlived its usefulness. We should probably switch to a more flexible scheme where sample apps, screenshots and this file can all be  updated by hand.

We should talk about exactly how to organize that change.

Reply:
I'm under the impression it's referring to https://flutter.io/catalog/samples/ rather than https://flutter.io/widgets/. 

Previous edits https://github.com/flutter/website/commits/master/_data/catalog/widgets.json seem like hand edits.

Reply:
Sorry, my mistake. My point about the sample catalog is spot-on, since even I've forgotten how the samples are organized, and I'm the one who organized them.